endpoints/openrtb2/auction_test.go

```diff
@@ -2889,183 +2889,6 @@ func TestValidateSourceTID(t *testing.T) {
 	assert.NotEmpty(t, req.Source.TID, "Expected req.Source.TID to be filled with a randomly generated UID")
 }
 
-func TestSChainInvalid(t *testing.T) {
```

Comment:
We should restore the prebid specific per-bidder schain logic as mentioned in another comment so this test should be restored as well.

Reply:
Restored